### DISCUSSION TITLE
Can MPD follow symlinks

### DISCUSSION THREAD

**Discussion by alwynpinto** [MusicPlayerDaemon/MPD#1802] [General]
I have created  a folder called "~/Music/servermusic"
and in the folder I have created symlinks to multiple folders on my server

in the mpd.conf i have added the above folder in the music_directory and added an additional line for symlinks
`
music_directory		"/home/hp/Music/servermusic"
follow_outside_symlinks "yes"
`
When I run a mpd client, it says permission denied. I have given chmod 777 privileges to servermusic and other folders in there.

What can I do to have multiple folders under music_directory. I assumed symlinks would work. Is there another way ?

**Comment by MaxKellermann** [comment]
https://www.musicpd.org/help/
